DEPRECATED.md

```diff
@@ -15,7 +15,36 @@ You can use the static helper methods such as `System::loadLanguageFile()` or
 The constants `TL_ROOT`, `TL_MODE`, `TL_START` and `TL_SCRIPT` have been
 deprecated and will be removed in Contao 5.0.
 
-// FIXME: show the alternatives
+Instead of `TL_ROOT` use:
+
+```php
+global $kernel;
+
+$rootDir = $kernel->getContainer()->getParameter('kernel.root_dir');
+```
+
+Instead of `TL_MODE` to check for `BE` and `FE` use:
+
+```php
+global $kernel;
+
+$isFE = $kernel->getContainer()->isScopeActive('frontend');
+$isBE = $kernel->getContainer()->isScopeActive('backend');
+```
+
+Note: You can also use the `ScopeAwareListener` in your own event listeners.
```

Comment:
I mean what is written. You can use the `ScopeAwareListener` in your own event listeners if you like.